src/OmnichannelChatSDK.ts

```diff
@@ -157,7 +159,19 @@ class OmnichannelChatSDK {
             RequestId: this.requestId
         });
 
-        if (optionalParams.liveChatContext) {
+        if (this.isPersistentChat) {
```

Comment:
I'd make sure this logic only works when persistent chat is explicitly enabled by changing the condition to `this.isPersistentChat && !this.chatSDKConfig.persistentChat.disable`